user/client.js

```diff
@@ -0,0 +1,13 @@
+'use strict'
+
+class UserClient {
+  constructor (userService) {
+    this._userService = userService
+  }
+
+  getMe (_id) {
+    return this._userService.getProfile(_id)
+  }
+}
```

Comment:
What's the reason to have this as a class?

Reply:
Not at all. But why not?

Reply:
it’s 10 lines more than what’s needed. It send the wrong message.

Reply:
Done